app/model/Common/File.php

```diff
@@ -31,6 +31,13 @@ public function getFileName() : string
         return basename($this->path->getPath());
     }
 
+    public function getFileNameWithoutPrefix() : string
```

Comment:
1. Nebylo by lepší `getOriginalFileName()`
2. Přidat test, že to opravdu vrací originální název.

Reply:
přejmenoval jsem to a přidal test do integracniho testu

Reply:
Ten test pls přidej přímo jako test do toho unit testu, který přibyde po mergnutí https://github.com/skaut/Skautske-hospodareni/pull/1175

Ať je to explicitně testovaný, než že failne nepřímo související test.

Reply:
Ten co vznikne je ale FilePath, zatimco tohle volam na File

Reply:
V tom případě přidat `FileTest`.